src/fragments/lib/analytics/native_common/getting-started/30_initAnalytics.mdx

```diff
@@ -13,4 +13,10 @@ import amplifyconfig from './amplifyconfiguration.json';
 Amplify.configure(amplifyconfig);
 ```
 
+<Callout warning="true">
```

Comment:
I know fragments are banned, but this seems like a perfect use for one -_-

Reply:
Yeah, I checked with René initially and he affirmed that we should not be using them even in this case.